File: QE-SELENIUM-JAVA/demo/src/main/java/demos/Activity5.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {
    public static void main(String[] args) throws InterruptedException {

        FirefoxDriver driver = new FirefoxDriver();  
        driver.get("https://training-support.net/webelements/dynamic-controls");

        System.out.println("Page Title: " + driver.getTitle());

        WebElement checkbox = driver.findElement(By.id("checkbox"));
        System.out.println("Checkbox displayed: " + checkbox.isDisplayed());

        driver.findElement(By.id("checkbox")).click();
        Thread.sleep(2000);

        System.out.println("checkbox displayed after remove: " + checkbox.isDisplayed());

        driver.quit();
    }
}

File: QE-SELENIUM-PYTHON/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/dynamic-controls")

print("Page Title:", driver.title)

checkbox = driver.find_element(By.ID, "checkbox")
print("Checkbox displayed:", checkbox.is_displayed())

driver.find_element(By.ID, "checkbox").click()
time.sleep(2)

print("checkbox displayed after remove:", checkbox.is_displayed())

driver.quit()